app/src/main/res/drawable/addon_textview_selector.xml

```diff
@@ -3,6 +3,6 @@
    -  License, v. 2.0. If a copy of the MPL was not distributed with this
    -  file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:state_enabled="true" android:color="?android:attr/textColorPrimary" />
-    <item android:state_checked="false" android:color="@color/photonGrey40" />
+    <item android:state_enabled="false" android:color="@color/photonGrey40" />
```

Comment:
cc @gabrielluong / @Amejia481 I see we are using a text color selector for the TextView text colors, but I don't see us setting the state of those views to disabled anywhere? Should they be disabled at some point? Or can we remove this selector?

Reply:
Thanks for pointing this up. I think at some point we were disabling these views, but now we are not using it anymore, I think it is safe to delete it as long as we remove all instances where it is referenced.